.github/workflows/publish.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Is your plan to release only when you manually push a tag?  The workflow you mentioned in slack published to PyPI after they [create a new release](https://github.com/dpguthrie/dbtc/blob/1e83a4031f4dc89c99ebd32de6d7dce222d5ec11/.github/workflows/publish.yml#L3-L5).  Either is fine, just want to be clear on when this gets triggered.

Core has a more complex design that tags it and then releases it for you to both GitHub and PyPI.

Reply:
<img width="453" alt="Screen Shot 2023-06-13 at 12 56 44 PM" src="https://github.com/dbt-labs/dbt-meshify/assets/53586774/3d87fbb6-a629-41f3-910d-1a16797acc2c">

I think the plan was to publish to PyPI when we create a new release. I might be confused here because I thought creating a new release _also_ involved creating a new tag on publish.

Reply:
Gotcha - sounds like release is the way to go then. Thanks for the explanation